src/components/MessagesList/MessagesGroup/Message/MessageButtonsBar/MessageButtonsBar.vue

```diff
@@ -475,7 +475,8 @@ export default {
 .message-buttons-bar {
 	display: flex;
 	right: 14px;
-	bottom: -4px;
+	top: 0px;
+	transform: translateY(-35%);
```

Comment:
```suggestion
	top: -16px;
```

I agree with you with the approach of anchoring the menu to the top of the message but why not just specify a negative top?

Reply:
Ok, thanks :)